src/munge.spec.ts

```diff
@@ -86,24 +89,67 @@ describe('munging', () => {
       expect(validateOfferCodecs(parsed)).toBe(true);
     });
   });
+
   describe('retainCodecs', () => {
     it('should retain codecs correctly when passing in an SDP', () => {
       expect.hasAssertions();
       const offer = fs.readFileSync('./src/sdp-corpus/offer_with_extra_codecs.sdp', 'utf-8');
       const parsed = parse(offer);
 
-      retainCodecs(parsed, ['h264', 'opus']);
+      // eslint-disable-next-line jsdoc/require-jsdoc
+      const predicate = (codecInfo: CodecInfo) =>
```

Comment:
If this predicate is the same for multiple tests, consider putting it in a describe to avoid duplicating code.

Reply:
It's only being used for half of the tests though. To me, it would make sense to do so if most or all of the tests use it (and in the future, if we add more cases to this describe block, that means most of the test cases will not use it).